Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -157,13 +157,14 @@ public extension String {
 
 	#if canImport(Foundation)
 	/// SwifterSwift: Check if string is valid email format.
+	/// **Note that this property does not validate the email address against an email server. It merely attempts to determine whether its format is suitable for an email address.**.
```

Comment:
You can use the `Note` annotation instead. I.e.
```
///
/// - Note:
///  Note that this property does not validate the email address against an email server. It merely attempts to determine whether its format is suitable for an email address.
///
```
That way it'll appear in bold in the callout.